.env.example

```diff
@@ -2,6 +2,9 @@
 TESTNET_ACCOUNT=
 MAINNET_ACCOUNT=
 
+#### sphinx
+SPHINX_API_KEY=
+
```

Comment:
The script calls for deploying on `optimism_sepolia`, but this file has not been updated to include an RPC URL for that network.